config/authorities/metadata_formats.yml

```diff
@@ -0,0 +1,7 @@
+terms:
+  - id: application/xml; schema=fgdc
+    term: FGDC
+  - id: application/xml; schema=iso19139
+    term: ISO19139
+  - id: application/xml; schema=mods
```

Comment:
I found a couple instances of `application/fgdc+xml`, should we go with that?  Maybe `application/iso19139+xml` too?